File: CarHomeWidget/src/main/java/com/anod/car/home/skin/GlossyProperties.java
package com.anod.car.home.skin;

import com.anod.car.home.R;

public class GlossyProperties extends BaseProperties {

    @Override
    public int getLayout(int number) {
        if (number == 4) {
            return R.layout.sk_glass_4;
        }
        if (number == 8) {
            return R.layout.sk_glass_8;
        }
        return R.layout.sk_glass_6;
    }

    @Override
    public int getSettingsButtonRes() {
        return R.drawable.ic_settings;
    }

    @Override
    public int getRowLayout() {
        return R.layout.sk_glass_row;
    }
}

File: CarHomeWidget/src/main/java/com/anod/car/home/skin/GlossyProperties.kt
package com.anod.car.home.skin

import com.anod.car.home.R

class GlossyProperties : BaseProperties() {

    override val settingsButtonRes: Int
        get() = R.drawable.ic_settings

    override val rowLayout: Int
        get() = R.layout.sk_glass_row

    override fun getLayout(number: Int): Int {
        if (number == 4) {
            return R.layout.sk_glass_4
        }
        return if (number == 8) {
            R.layout.sk_glass_8
        } else R.layout.sk_glass_6
    }
}
